rust/fastsim-core/src/simdrivelabel.rs

```diff
@@ -704,7 +709,7 @@ pub fn get_label_fe_phev(
         };
     }
 
-    Ok(phev_calcs)
+    return Ok(phev_calcs);
```

Comment:
The idiomatic way to return in rust is `Ok(phev_calcs)` so please change it back to `Ok(phev_calcs)` since it will otherwise give warnings whenever we run `cargo clippy`.